backends/cxxrtl/runtime/cxxrtl/cxxrtl_replay.h

```diff
@@ -556,22 +556,20 @@ class recorder {
 	bool record_incremental(ModuleT &module) {
 		assert(streaming);
 
-		struct : public observer {
```

Comment:
I take it user-defined observers are now not supposed to derive from the generic `observer` struct.

Reply:
You can do either. I removed the base class in this particular struct so that I can capture using an aggregate initializer; this is purely a consequence of a stylistic choice elsewhere.